src/coreComponents/common/initializeEnvironment.cpp

```diff
@@ -201,6 +205,19 @@ void setupCaliper( cali::ConfigManager & caliperManager,
 #endif
   adiak::value( "CUDA runtime version", cudaRuntimeVersion );
   adiak::value( "CUDA driver version", cudaDriverVersion );
+
+  // HIP info
+  int hipRuntimeVersion = 0;
+  int hipDriverVersion = 0;
```

Comment:
Should these be set?

Reply:
This is just mimicking the same calls for cuda just above for consistency. If we want to change both we can address it in a quick follow-up.